### PR TITLE
NGSTACK-855 notifications module

### PR DESCRIPTION
**Pull request by PetraMiseta**
Added notifications module in backoffice

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -54,7 +54,8 @@
         "netgen/layouts-ibexa-site-api": "~1.4.0",
         "netgen/layouts-ibexa-relation-list-query": "^1.4",
         "netgen/layouts-ibexa-tags-query": "^1.4",
-        "netgen/conversations": "^1.0"
+        "netgen/conversations": "^1.0",
+        "netgen/notifications": "*"
```
Comment (iherak) [inline composer.json:58]: We should limit this to a version, probably same as `netgen/conversations` above.

File: config/packages/netgen_notifications.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    categories:
+        default:
+            subscription:
+                enabled: true
+                resolver: netgen_notifications.ibexa.subtree_update_resolver
+                digest_actions:
+                    email:
+                        type: digest_email
+                        parameters:
+                            from: 'info@netgen.io'
```
Comment (iherak) [inline config/packages/netgen_notifications.yaml:11]: This should be a parameter (env variable?)

File: config/packages/netgen_notifications.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+                            subject: 'Here is what you missed'
+                            template: backoffice/notifications/digest.html.twig
+            actions:
+                email:
+                    type: email
+                    parameters:
+                        from: 'info@netgen.io'
```
Comment (iherak) [inline config/packages/netgen_notifications.yaml:18]: Same as above.

File: config/packages/netgen_notifications.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+        default:
+            subscription:
+                enabled: true
+                resolver: netgen_notifications.ibexa.subtree_update_resolver
+                digest_actions:
+                    email:
+                        type: digest_email
+                        parameters:
+                            from: 'info@netgen.io'
+                            subject: 'Here is what you missed'
```
Comment (iherak) [inline config/packages/netgen_notifications.yaml:12]: I am not sure of the scope of work here, but the subject should probably be translateable?

Comment (emodric) [reply config/packages/netgen_notifications.yaml:12]: Not possible currently. Limitation of the notifications bundle.

File: composer.json (hunk 1 above)
Comment (PetraMiseta) [reply composer.json]: Done - https://github.com/netgen/media-site/pull/189/commits/cdac9340011ec2bd393e2fe00b1002b235e96bf0

File: config/packages/netgen_notifications.yaml (hunk 2 above)
Comment (PetraMiseta) [reply config/packages/netgen_notifications.yaml]: Done - https://github.com/netgen/media-site/pull/189/commits/3a4cdb49b1d4770ba552b43bbdb3f79da497ed4f

File: config/packages/netgen_notifications.yaml (hunk 3 above)
Comment (PetraMiseta) [reply config/packages/netgen_notifications.yaml]: Done - https://github.com/netgen/media-site/pull/189/commits/3a4cdb49b1d4770ba552b43bbdb3f79da497ed4f

**Review by iherak — APPROVED** [review]

**Review by emodric — APPROVED** [review]

**Pull request merged**